sdks/python/apache_beam/transforms/external.py

```diff
@@ -694,12 +695,31 @@ class JavaJarExpansionService(object):
   def __init__(self, path_to_jar, extra_args=None, classpath=None):
     self._path_to_jar = path_to_jar
     self._extra_args = extra_args
-    self._classpath = classpath
+    self._classpath = classpath or []
     self._service_count = 0
 
+  @staticmethod
+  def _expand_jars(jar):
+    if glob.glob(jar):
+      return glob.glob(jar)
+    elif isinstance(jar, str) and (jar.startswith('http://') or
+                                   jar.startswith('https://')):
+      return [jar]
+    else:
+      try:
```

Comment:
Please add a comment to explain the "else" clause here.

Reply:
done